docs/_blog/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Command Line intro to Bitcoin Wallet Dev"
+description: "Intro to bdk-cli and wallet dev"
+authors:
+    - waterst0ne Pedro Felix
+date: "2022-09-22"
+tags: ["bdk-cli", "basics","novice"]
+permalink: "/_blog/bdk_cli_basics_00"
+---
+
+# Bicoin Dev Kit - Command Line Intro to Bitcoin Wallet Development 
```

Comment:
This should read something like "Tutorial Goals".. The description for the tutorial is already there in the title.. Also I feel marking it with ## instead # would look more nice in symmetry with other chapter titles.

Reply:
Thank you.